package.json

```diff
@@ -6,7 +6,7 @@
     "autoprefixer": "6.4.1",
     "babel-core": "6.14.0",
     "babel-eslint": "6.1.2",
-    "babel-jest": "15.0.0",
+    "babel-jest": "^19.0.0",
```

Comment:
This was added automaticaly by yarn, if it can be cause of some problems I can remove it.